### ISSUE TITLE
Move along

### ISSUE THREAD

**Issue by n1kCS** [Maxx53/SCMBot#359]
hi i just recently started using SCM and ive been getting this error everytime
"Move along...."

**Comment by anileger** [comment]
+1.

But I know that Max will come here and say again " I don't give a f.ck. ". nvm.

@Maxx53  steam has been updated recently. I think something is wrong with that new patch.

**Comment by Jentosss** [comment]
Nothing changed. Don't listen to them Max. Bot is working perfectly.

**Comment by vampig** [comment]
Jentosss, shhh! Bot does not work, and never worked! ._.

**Comment by anileger** [comment]
@jentosss You are wrong. Either bot is working or not. It depends on time. I got "move along" alert many times on a day but bot also could buy an item. I said its all about time. @Maxx53 wont give a f.ck but I have to say that bug. HF

**Comment by Maxx53** [comment]
@anileger high five, bro :hand:

**Comment by anileger** [comment]
@Maxx53 high five, dude :smirk:
